recipes/aldjemy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "aldjemy" %}
+{% set version = "0.6.0"%}
+{% set sha256 = "a0f8e684d5c7b97dff64d0da0d75ca6a1f3062d6eca9e54c188e7babbe2439fa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sqlalchemy >0.7.1
```

Comment:
I don't think `sqlalchemy` is a build requirement of this package. Please remove it.